predictions.py

```diff
@@ -13,9 +19,12 @@ def loop(request_q, response_q):
     the response_q queue.
     """
     # just leaving these here for future reference (elmiko)
-    # from pyspark import sql as pysql
-    # spark = pysql.SparkSession.builder.appName("JiminyRec").getOrCreate()
-    # sc = spark.sparkContext
+    from pyspark import sql as pysql
```

Comment:
originally i had put this import here to help remember we needed it, i think we should move it to the top level imports for the module.